index.js

```diff
@@ -249,4 +252,28 @@ class Analytics {
   }
 }
 
+const retryCondition = function (error) {
```

Comment:
```js
const retryCondition = error => {
  // ...
}
```

Reply:
thanks! roland just explained what this was to me :)

Reply:
Roland actually suggested not assigning and using it as a function variable at all, which makes sense to me.

```
function _retryCondition(error) {
```

Reply:
I don't follow, but either way, we need to expose this logic somehow in order to test it.

Reply:
> Roland actually suggested not assigning and using it as a function variable at all, which makes sense to me.

Why not?

Reply:
To me `function _retryCondition(error) {` looks cleaner than `const _retryCondition = error => {`. 🤷‍♂️

Reply:
🚲 🏠